lib/helpers/schema-utils.js

```diff
@@ -35,14 +35,20 @@ module.exports.getSchemaByFilesPath = (apiDefinitionsPath, buildSchemaOptions) =
 };
 
 module.exports.getValidatorByPathMethodAndCode = (schema, request, response) => {
-  const route = pathMatcher(schema, request.path);
+  const route = pathMatcher(schema, request.path, request.method);
   return get(schema, `${route}.${request.method}.responses.${response.status}`);
 };
 
 module.exports.pathMatcher = pathMatcher;
-function pathMatcher(routes, path) {
+function pathMatcher(routes, path, method) {
   return Object
```

Comment:
lines 44-51 should be extracted and run once per routes, no need to sort on each call